vendor/k8s.io/kubernetes/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -81,9 +86,14 @@ func NewOrderedKubeAdmissionPlugins(kubeAdmissionOrder []string) []string {
 	ret := []string{}
 	for _, curr := range kubeAdmissionOrder {
 		if curr == mutatingwebhook.PluginName {
-			ret = append(ret, openshiftAdmissionPluginsForKube...)
+			ret = append(ret, openshiftAdmissionPluginsForKubeBeforeMutating...)
 			ret = append(ret, customresourcevalidationregistration.AllCustomResourceValidators...)
 		}
+		if curr == resourcequota.PluginName{
+			ret = append(ret, curr)
+			ret = append(ret, openshiftAdmissionPluginsForKubeAfterResourceQuota...)
```

Comment:
move that block below the append below. Now the flow is complicated.

Reply:
done